ament_copyright.json

```diff
@@ -0,0 +1,16 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "ament_copyright",
+            "fileLocation": ["absolute"],
+            "pattern": [
+                {
+                    // /.../sqlite3_vendor/cmake/Modules/FindSQLite3.cmake: could not find copyright notice
```

Comment:
JSON doesn't support comments.

Reply:
This is MS own variant of it, so it works:
https://code.visualstudio.com/Docs/editor/tasks#_defining-a-problem-matcher

Reply:
removed!